appinfo/app.php

```diff
@@ -2,3 +2,4 @@
 OC::$CLASSPATH['OC_User_IMAP']='user_external/lib/imap.php';
 OC::$CLASSPATH['OC_User_SMB']='user_external/lib/smb.php';
 OC::$CLASSPATH['OC_User_FTP']='user_external/lib/ftp.php';
+OC::$CLASSPATH['OC_User_FTP']='user_external/lib/xmpp.php';
```

Comment:
the class here should probably be `OC_User_XMPP`

Reply:
implemented.